lib/testsuite/index.js

```diff
@@ -503,6 +532,10 @@ class TestSuite {
   }
 
   async startTestSuite() {
+    NightwatchEventHub.emit(TestSuiteHook.started, {
+      testResults: this.reporter.testResults.export
```

Comment:
@beatfactor I think we should not sent the entire report in every event. I think we should create a new format to send only the data that is relevant for a particular event. What do you think?

Reply:
it would indeed be better to send only the necessary data.